webserver/src/main/java/com/mlaide/webserver/model/Project.java

```diff
@@ -1,16 +1,22 @@
 package com.mlaide.webserver.model;
 
+import com.mlaide.webserver.validation.ValidationRegEx;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Pattern;
 import java.time.OffsetDateTime;
 
 @Getter
 @Setter
 @NoArgsConstructor
 public class Project {
+    @NotBlank
     private String name;
```

Comment:
Eigentlich ist name kein Pflichtfeld. In ProjectServiceImpl prüfen wir ob name gesetzt ist. Falls nicht wird einfach der key als name verwendet. Beide Möglichkeiten sind in Ordnung. Aber wir sollten uns für eine der beiden entscheiden und entsprechend den anderen Code entfernen.

Spielt auch nur ne Rolle aus dem Angular Frontend. Über den Python Client kann man keine Projekte anlegen.

Reply:
Im Frontend + Swagger ist es aktuell ein Pflichtfeld, würde das auch so lassen. Ich entferne den Automatismus.